topic_tools/CMakeLists.txt

```diff
@@ -87,6 +87,20 @@ target_link_libraries(mux
 if(BUILD_TESTING)
   find_package(ament_lint_auto REQUIRED)
   ament_lint_auto_find_test_dependencies()
+
+  find_package(ament_cmake_gtest REQUIRED)
+  ament_add_gtest(test_relay test/test_relay.cpp)
+  target_include_directories(test_relay PUBLIC
+    $<BUILD_INTERFACE:${CMAKE_CURRENT_SOURCE_DIR}/include>
+    $<INSTALL_INTERFACE:include>
+  )
+  ament_target_dependencies(test_relay
+    rclcpp
+    std_msgs
```

Comment:
you'll need to add `find_package(std_msgs REQUIRED) above and add `std_msgs` it as a `<test_depend>` in the `package.xml` file

Reply:
Thanks! Done.